DLLVersion.rc

```diff
@@ -4,8 +4,8 @@
 
 #include "winres.h"
 
-#define POCO_VERSION 1,13,1,0
-#define POCO_VERSION_STR "1.13.1"
+#define POCO_VERSION 1,13,2,0
```

Comment:
shouldn't this be `1,13,3,0`?

Reply:
Version 1.13.3 was released with this wrong version in the rc file.

Reply:
I re-released it this morning, hoping that none of the packagers has yet used the faulty packages ;-)